guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+---
+layout: guide
+title: Design best practices
+description: An overview on best design practices for bitcoin-backed ecash.
+nav_order: 5
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/design-best-practices
+redirect_from:
+ - /guide/how-it-works/ecash/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/best-practicies.jpg
+image_base: /assets/images/guide/how-it-works/ecash/
+images_mint-list:
+    - file: federation-list
+      modalImage: federation-list@2x
+      alt: List of federations with QR codes for easy scanning and joining.
+      caption: Federations with QR codes. Each QR code contains the federation's URL, allowing users to display, and invite another person to scan and join.
+    - file: cashu-mint-list
+      modalImage: cashu-mint-list@2x
+      alt: List of Cashu mints with mint addresses displayed.
+      caption: Cashu mints with mint addresses displayed.
+
+images_pending:
+    - file: pending-token
+      modalImage: pending-token@2x
+      alt: A transaction history that shows received, pending, and sent transaction each with their own visual indicator.
+      caption: Received, pending, and sent transaction each with their own visual indicator.
+    - file: pending-transaction-details
+      modalImage: pending-transaction-details@2x
+      alt: Expanded view of a pending transaction, showing the memo, amount, and the mint that created the token.
+      caption: Expanded view of a pending transaction. While token is pending, anyone with the token string can claim it.
+    - file: pending-transaction-qr
+      modalImage: pending-transaction-qr@2x
+      alt: A QR code for a pending transaction with the token string available to copy to clipboard.
+      caption: QR code for a pending transaction.
+
+images_mint-default:
+    - file: cashu-mint-details
+      modalImage: cashu-mint-details@2x
+      alt: Cashu Mint metadata fields displayed in a wallet interface.
+      caption: Cashu Mint metadata fields displayed in a wallet interface.
+    - file: cashu-mint-details-settings
+      modalImage: cashu-mint-details-settings@2x
+      alt: A mobile ecash wallet with a bottom sheet that lists the following actions, refresh mint settings, update mint url, set as default mint, and remove mint.
+      caption: Advance mint operations like setting a default mint, updating mint url, and removing a mint.
+    - file: cashu-mint-details-default-set-confirmation
+      modalImage: cashu-mint-details-default-set-confirmation@2x
+      alt: A success confirmation that the mint has been set as default.
+      caption: Confirmation when a mint has been set as default.
+    - file: cashu-default-applied
+      modalImage: cashu-default-applied@2x
+      alt: A list of mints with the default mint highlighted by a home icon.
+      caption: Default mint highlighted by a home icon.
```

Comment:
Different wallets handle the home screen balance in various ways. If you setup a default mint, in my opinion it would make sense to show only the balance the user has on that mint because the user has selected a default.

How other wallets handle home screens;
Minibits only shows your balance for a specific mint, and you have to swipe right to view each mint and its unique balance. You can also go to a mint overview screen to see all your balances across different mints.

Cashu.Me and eNuts display your entire collective balance across all mints on the home screen. Cashu.Me shows two balances: a 'grand total' and a 'currently select mint balance'. eNuts doesn't ever show an individual mint balance, even if you assign a default mint. It also shows your total balance across all mints. On eNuts, if you want to see your balances across all mints, you need to go to the mint settings, where a list is displayed.

I think all these solutions could work. Personally, I’m not convinced any one method is better than another. As long as it’s clear what the user is looking at, I believe it comes down to personal preference.